src/Microsoft.Azure.WebJobs.Host/JobHost.cs

```diff
@@ -23,7 +24,7 @@ namespace Microsoft.Azure.WebJobs
     /// A <see cref="JobHost"/> is the execution container for jobs. Once started, the
     /// <see cref="JobHost"/> will manage and run job functions when they are triggered.
     /// </summary>
-    public class JobHost : IDisposable
+    public class JobHost : IDisposable, IServiceProvider
```

Comment:
Moved it to a property.

---
In reply to: [124319474](https://github.com/Azure/azure-webjobs-sdk/pull/1202#discussion_r124319474) [](ancestors = 124319474)